littleballoffur/backend.py

```diff
@@ -22,77 +23,66 @@ def get_number_of_nodes(self, graph: NKGraph) -> int:
         """
         return graph.numberOfNodes()
 
-
```

Comment:
@benedekrozemberczki sorry for formatting getting mixed into this PR but this is a PEP thing for methods within a class.